src/estimagic/optimization/fides_optimizers.py

```diff
@@ -0,0 +1,251 @@
+"""Implement the fides optimizer."""
+import logging
+from functools import partial
+
+import numpy as np
+from estimagic.config import IS_FIDES_INSTALLED
+from estimagic.optimization.algo_options import CONVERGENCE_ABSOLUTE_CRITERION_TOLERANCE
+from estimagic.optimization.algo_options import CONVERGENCE_ABSOLUTE_GRADIENT_TOLERANCE
+from estimagic.optimization.algo_options import CONVERGENCE_ABSOLUTE_PARAMS_TOLERANCE
+from estimagic.optimization.algo_options import CONVERGENCE_RELATIVE_CRITERION_TOLERANCE
+from estimagic.optimization.algo_options import CONVERGENCE_RELATIVE_GRADIENT_TOLERANCE
+from estimagic.optimization.algo_options import STOPPING_MAX_ITERATIONS
+
+if IS_FIDES_INSTALLED:
+    from fides import hessian_approximation
+    from fides import Optimizer
+
+
+def fides(
+    criterion_and_derivative,
+    x,
+    lower_bounds,
+    upper_bounds,
+    *,
+    hessian_update_strategy="bfgs",
+    convergence_absolute_criterion_tolerance=CONVERGENCE_ABSOLUTE_CRITERION_TOLERANCE,
+    convergence_relative_criterion_tolerance=CONVERGENCE_RELATIVE_CRITERION_TOLERANCE,
+    convergence_absolute_params_tolerance=CONVERGENCE_ABSOLUTE_PARAMS_TOLERANCE,
+    convergence_absolute_gradient_tolerance=CONVERGENCE_ABSOLUTE_GRADIENT_TOLERANCE,
+    convergence_relative_gradient_tolerance=CONVERGENCE_RELATIVE_GRADIENT_TOLERANCE,
+    stopping_max_iterations=STOPPING_MAX_ITERATIONS,
+    stopping_max_seconds=np.inf,
+    trustregion_initial_radius=1.0,
```

Comment:
Don't we have a default for initial trust region radius?

Reply:
I did not find it in `algo_options.py` so I decided to hard code the Fides default.